server/controllers/userController.js

```diff
@@ -0,0 +1,51 @@
+import userHelper from '../helpers/UserHelper';
+import authHelper from '../helpers/authHelpers';
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * User routes.
+ */
+class userController {
+  /**
+   * This method handle the update profile request.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the user.
+   */
+  static async updateProfile(req, res) {
+    const { token } = req.headers;
+    const {
+      lastname,
+      firstname,
+      gender,
+      birthdate,
+      preferredlanguage,
+      preferredcurrency,
+      location,
+      Department,
+      emailNotifications,
+      inAppNotifications
+    } = req.body;
+    const { id } = await authHelper.verifyToken(token);
+    const data = {
+      lastname,
+      firstname,
+      gender,
+      birthdate,
+      preferredlanguage,
+      preferredcurrency,
+      location,
+      Department,
+      emailNotifications,
+      inAppNotifications
+    };
```

Comment:
Leave an empty space after the semi-colon.

Reply:
done